### DISCUSSION TITLE
建议添加“直播间自动最高画质”和“直播间自动网页全屏”两个组件

### DISCUSSION THREAD

**Discussion by fate1ess** [the1812/Bilibili-Evolved#3421] [功能建议 / Ideas]
建议添加“直播间自动最高画质”和“直播间自动网页全屏”两个组件。

**Comment by Geeyun-JY3** [comment]
重复：
- 「直播间自动最高画质」相关：
  - 默认画质：#1324、#1515、#1536、#1899、#2364、#2881、#3188
  - 自动（默认）最高画质（原画）：#1660、#2682、#2853、#2883、#3065
- 「直播间自动网页全屏」相关：#965
#1019、<https://github.com/the1812/Bilibili-Evolved/discussions/1994#discussioncomment-924116>、#2439、#3079

而且你这开了又开是干啥……：#1019、#2439

**Reply by fate1ess** [reply]
我都忘了以前问过，不好意思。关键现在我在组件里面没找到这两个设置啊，搜不到，然后翻了一遍所有组件还是没找到。。请问在哪里？

**Reply by Geeyun-JY3** [reply]
没说做了啊，你认真看看最早几个的 issue 和部分 discussion 被打的 label。
